exporter/awsprometheusremotewriteexporter/auth.go

```diff
@@ -73,14 +69,20 @@ func (si *signingRoundTripper) RoundTrip(req *http.Request) (*http.Response, err
 }
 
 func newSigningRoundTripper(auth AuthConfig, next http.RoundTripper) (http.RoundTripper, error) {
+	if auth.Region == "" {
+		// TODO(jbd): Automatically parse the region from the workspace.
+		return next, nil
+	}
+	if auth.Service == "" {
```

Comment:
Shouldn't we ignore the user input, log a warning, and remove the configuration by 1.0?

Reply:
I suggest keeping this option for AWS engineers in case they want to test it with some other service. Otherwise, end users never need to set it up.